src/main/java/io/arlas/server/wfs/filter/ElasticAttribute.java

```diff
@@ -0,0 +1,223 @@
+/**
```

Comment:
There is a License paradox.
I guess we should remove Public Domain disclaimer and only display ALv2.

Reply:
fix